sdk/core/azure-core-test/src/main/java/com/azure/core/test/InterceptorManager.java

```diff
@@ -331,7 +332,10 @@ public HttpClient getPlaybackClient() {
                 throw new IllegalStateException("A playback client can only be requested in PLAYBACK mode.");
             }
             if (testProxyPlaybackClient == null) {
-                testProxyPlaybackClient = new TestProxyPlaybackClient(httpClient, skipRecordingRequestBody);
+//                reactor.netty.http.client.HttpClient client = reactor.netty.http.client.HttpClient.create().wiretap("reactor.netty.http.client.HttpClient", LogLevel.TRACE, AdvancedByteBufFormat.TEXTUAL);
```

Comment:
Did you want to remove this commented out code or if not add another comment above explaining when someone would want to make a swap here

Reply:
Definitely. This is really a draft PR at this point as we continue to chase down the problems. I'll clean a lot of things up when I'm done.